lib/joystick.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const {
+  joystickIsConnected: isConnected,
+  joystickGetButtonCount: getButtonCount,
+  joystickHasAxis: hasAxis,
+  joystickIsJoystickButtonPressed: isButtonPressed,
+  joystickGetAxisPosition: getAxisPosition,
+  joystickGetIdentification: getIdentification,
+  joystickUpdate: update,
+  joystickAxis: Axis,
+} = require('../build/Release/binding.node');
+
+module.exports.isConnected = isConnected;
```

Comment:
We usually do not export method directly if it has any parameter(s). Because we should validate all the parameter(s) in JavaScript side to avoid any coredump.

e.g.

- We export `Mouse.isButtonPressed()` directly because it has no parameter.
- We wrap `Mouse.getPosition()` because it has a parameter. And we should ensure the parameter passed into native code is certainlly right.

Reply:
```js
console.log(Joystick.getIdentification(-1));
```

will occur a coredump.

And if we do the validation in C++ side, it will be complex.

Reply:
I will add JS checks. I though the Nan thing was doing those checks and the error reporting, my bad ^^